lib/gui/lvgl/Kconfig

```diff
@@ -216,6 +216,15 @@ config LVGL_SCREEN_REFRESH_PERIOD
 	help
 	  Screen refresh period in milliseconds
 
+config LVGL_INPUT_POINTER
+	bool "Enable input pointer device"
+
+config LVGL_INPUT_POINTER_DEV_NAME
+	string "Input pointer device name"
+	default "INPUT"
```

Comment:
Maybe a `depends on LVGL_INPUT_POINTER` would make sense here, so that this symbol doesn't show up in the menu when `LVGL_INPUT_POINTER` is disabled.